packages/genesys-spark-components/src/components/stable/gux-icon/icon-names-deprecated.ts

```diff
@@ -1,79 +1,433 @@
+  'control-stop': 'fa/stop-regular',
+  copy: 'fa/copy-regular',
+  'custom/add-column-after': 'custom/table-column-plus-right-regular',
+  'custom/add-column-before': 'custom/table-column-plus-left-regular',
+  'custom/add-row-after': 'custom/table-row-plus-bottom-regular',
+  'custom/add-row-before': 'custom/table-row-plus-top-regular',
+  'custom/binoculars-slash-regular': 'regular-binoculars-slash',
+  'custom/carousel': 'custom/rectangle-carousel-regular',
+  'custom/clipboard-list-slash-regular': 'regular-clipboard-list-slash',
+  'custom/co-browse': 'custom/screen-monitor-regular',
+  'custom/delete-column': 'custom/table-column-xmark-regular',
+  'custom/delete-row': 'custom/table-row-xmark-regular',
+  'custom/delete-table': 'custom/table-xmark-regular',
+  'custom/file-pen-question': 'custom/file-pen-question-regular',
+  'custom/filter-notification': 'custom/filter-dot-regular',
+  'custom/font-regular-highlight': 'custom/font-underline-regular',
+  'custom/one-button': 'custom/mobiles-one-button-regular',
+  'custom/person-running': 'fa/person-running-solid',
+  'custom/quick-reply': 'custom/circle-buttons-regular',
+  'custom/three-button': 'custom/mobile-three-button-regular',
+  'custom/wrap-text-inline': 'custom/wrap-text-inline-regular',
+  'custom/wrap-text-tight': 'custom/wrap-text-tight-regular',
+  'custom/wrap-text-top-bottom': 'custom/wrap-text-top-bottom-regular',
+  'customer-journey': 'fa/map-location-dot-regular',
+  cut: 'fa/scissors-regular',
+  dash: 'fa/gauge-simple-high-regular',
+  delete: 'fa/trash-regular',
+  'device-camera': 'fa/camera-regular',
+  'device-cellphone': 'fa/mobile-regular',
+  'device-headphones': 'fa/headphones-regular',
+  'device-laptop': 'fa/laptop-regular',
+  'device-pc': 'deprecated/device-pc',
+  'device-server': 'fa/server-regular',
+  'device-tablet': 'fa/tablet-regular',
+  download: 'fa/arrow-down-to-line-regular',
+  edge: 'fa/edge-legacy',
+  edit: 'fa/pen-regular',
+  'email-notification-score': 'fa/envelope-dot-regular',
   expand: 'fa/arrow-up-right-and-arrow-down-left-from-center-regular',
+  export: 'fa/arrow-up-from-line-regular',
+  'external-link': 'fa/arrow-up-right-from-square-regular',
+  'fa/00-solid': 'fa/00-regular',
+  'fa/arrows-rotate-solid': 'fa/arrows-rotate-regular',
+  'fa/ban-regular': 'fa/ban-regular',
+  'fa/check-solid': 'custom/check-small-regular',
+  'fa/circle-check-regular': 'fa/circle-check-regular',
+  'fa/circle-xmark-regular': 'fa/circle-xmark-regular',
+  'fa/comment-dots-regular': 'fa/message-dots-regular',
+  'fa/comments-regular': 'fa/messages-regular',
+  'fa/diagram-project-gear-regular': 'fa/regular-diagram-project-gear',
+  'fa/eye-regular': 'fa/eye-regular',
+  'fa/eye-slash-regular': 'fa/eye-slash-regular',
+  'fa/life-ring-regular': 'fa/life-ring-regular',
+  'fa/link-horizontal-slash-regular': 'fa/link-slash-regular',
+  'fa/link-horizontal-solid': 'fa/link-regular',
+  'fa/list-radio-clean-regular': 'fa/list-ul-regular',
```

Comment:
These icons are different so is this a new deprecation?

Reply:
This was done in a previous PR so assume it is correct